### PR TITLE
Pentest fix: first fix command injection in pickle.load()

### PR DESCRIPTION
**Pull request by Romanticoseu**
## Description

To fix pentest issue "Command Injection in pickle.load()", add verification before deserializing. Refer to this guide https://www.synopsys.com/blogs/software-security/python-pickling/.

Mainly referring to [pr8447](https://github.com/intel-analytics/BigDL/pull/8447)

### CODE REVIEW

File: python/friesian/src/bigdl/friesian/utils/safepickle.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+# Refer to this guide https://www.synopsys.com/blogs/software-security/python-pickling/
+# To safely use python pickle
+
+
+class SafePickle:
+    key = b'shared-key'
+    """
+    Example:
+        >>> from bigdl.chronos.utils import SafePickle
```
Comment (plusbang) [inline python/friesian/src/bigdl/friesian/utils/safepickle.py:30]: maybe change to `from bigdl.friesian.utils import SafePickle`, other `safepickle.py` is similar.

**Review by plusbang — COMMENTED** [review]
other LGTM, but please @lalalapotter also review to double check.

File: python/friesian/src/bigdl/friesian/utils/safepickle.py (hunk 1 above)
Comment (Romanticoseu) [reply python/friesian/src/bigdl/friesian/utils/safepickle.py:30]: have update

**Review by lalalapotter — APPROVED** [review]
LGTM

**Pull request merged**